libraries/botbuilder-lg/src/extractor.ts

```diff
@@ -35,15 +35,15 @@ export class Extractor
      * Extracts the templates and returns a map with their names and bodies.
      * @returns Map object with template names and bodies.
      */
-    public extract(): Map<string, any>[] {
-        const result: Map<string, any>[] = [];
-        this.templates.forEach((template: Template): any => {
-            const templateResult: Map<string, any> = new Map<string, any>();
+    public extract(): Map<string, string[] | Map<string, string[]>>[] {
+        const result: Map<string, string[] | Map<string, string[]>>[] = [];
+        this.templates.forEach((template: Template) => {
+            const templateResult: Map<string, string[] | Map<string, string[]>> = new Map<string, string[] | Map<string, string[]>>();
```

Comment:
Definitions like these can be shortened from `const result: Map<T, U> = new Map<T, U>()` to just `const result = new Map<T, U>()`. No need to duplicate the type.

Reply:
Note: this is just a suggestion for the future, no need to go back and revise all this code since most of it already uses the longer form.